### PR TITLE
WIP: Feature/global exception handler

### PR DESCRIPTION
**Pull request by srdjanstanojcic**
Global exception handler with testing from all layers.

### CODE REVIEW

File: BE/Business.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Business.Access.Layer.Helpers.GlobalExceptionHandler
+{
+    internal class AppException : Exception
```
Comment (Laban1992) [inline BE/Business.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs]: Class name should be more descriptive. Currently its no different from Exception class.

File: BE/Data.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Access.Layer.Helpers.GlobalExceptionHandler
+{
+    internal class AppException : Exception
```
Comment (Laban1992) [inline BE/Data.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs]: Why do we have two identical classes? Could they be centralized in a meaningful way?

File: BE/Business.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs (hunk 1 above)
Comment (srdjanstanojcic) [reply BE/Business.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs]: I made it just for testing purposes, this is a base code for global exception handler.

File: BE/Data.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs (hunk 2 above)
Comment (srdjanstanojcic) [reply BE/Data.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs]: It's  a base, I think they won't be same in future, but anyway I just made it like this to test if I can catch exception from business/data layer in api layer.

**Pull request closed without merging**